src/main/java/org/kohsuke/github/GHFork.java

```diff
@@ -0,0 +1,38 @@
+package org.kohsuke.github;
+
+/**
+ * The enum for Fork search mode
+ */
+public enum GHFork {
```

Comment:
This is copied from https://github.com/hub4j/github-api/blob/bcb71a3c318d243c92b757310cf153b8afc983d5/src/main/java/org/kohsuke/github/GHRepositorySearchBuilder.java#L219-L253 right?